pkg/automationconfig/automation_config_builder.go

```diff
@@ -274,17 +274,11 @@ func (b *Builder) Build() (AutomationConfig, error) {
 		},
 	}
 
-	if b.tlsConfig != nil && b.sslConfig != nil {
```

Comment:
But this will also allow for both SSL and TLS to be defined, is that correct?

Reply:
yes that's everything this change does